docs/using.rst

```diff
@@ -181,6 +181,19 @@ To use ADB with Host-Based tools
 .. _Docker Tooling: http://www.eclipse.org/community/eclipse_newsletter/2015/june/article3.php
       **Note:** Testing has been done with Eclipse 4.5.0.
 
+Using the ADB behind proxy
+==========================
+
+Today, Most of organizations have proxy enabled for internal network and good news is ADB can be setup behind proxy. All you need to do is un-comment proxy variable in the Vagrantfile and assign required value.
```

Comment:
s/ Today, Most of organizations have proxy enabled for internal network and good news is ADB can be setup behind proxy/ADB can be setup behind networks where proxy setup is required to access internet/